function-logs/handler.go

```diff
@@ -28,6 +32,16 @@ func Handle(req []byte) string {
 			if len(userQuery) > 0 {
 				user = userQuery
 			}
+			numLines = vals.Get("numLines")
+			sinceStr := vals.Get("since")
+			if sinceStr != "" {
+				var err error
+				since, err = strconv.Atoi(sinceStr)
```

Comment:
`since, err := strconv.Atoi(sinceStr)`
then you dont need to declare var err error.

Reply:
We need since outside this function, thats why i didnt do it using :=